From a299afaa354d8d795bf804dd3a8d381cc2e93591 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:53:04 -0400
Subject: [tools] perf/doc: Remove mention of non-existent set_perf_event_pending() from design.txt

Message-id: <1408953210-26343-297-git-send-email-jolsa@redhat.com>
Patchwork-id: 88984
O-Subject: [PATCH RHEL7.1 BZ1133083 296/322] perf/doc: Remove mention of non-existent set_perf_event_pending() from design.txt
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 0d4dd797564cddc1f71ab0b239e9ea50ddd40b2a
Author: Baruch Siach <baruch@tkos.co.il>
Date: Sat Jan 25 23:50:23 2014 +0200

description
===========
set_perf_event_pending() was removed in e360adbe ("irq_work: Add
generic hardirq context callbacks").
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/design.txt b/tools/perf/design.txt
index 67e5d0c..63a0e6f 100644
--- a/tools/perf/design.txt
+++ b/tools/perf/design.txt
@@ -454,7 +454,6 @@ So to start with, in order to add HAVE_PERF_EVENTS to your Kconfig, you
 will need at least this:
  - asm/perf_event.h - a basic stub will suffice at first
  - support for atomic64 types (and associated helper functions)
- - set_perf_event_pending() implemented
 
 If your architecture does have hardware capabilities, you can override the
 weak stub hw_perf_event_init() to register hardware counters.
-- 
1.7.1